go/kbnm/findbin.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/kardianos/osext"
+)
+
+var errKeybaseNotFound = errors.New("failed to find the keybase binary")
+
+// findKeybaseBinary returns the path to the Keybase binary, if it finds it.
+func findKeybaseBinary() (string, error) {
+	// Is it near the kbnm binary?
+	dir, err := osext.ExecutableFolder()
```

Comment:
Ah ok, this is "the directory that kbnm is in" rather than "the folder where executables live on this machine"?